### DISCUSSION TITLE
Multiple Solcast sites

### DISCUSSION THREAD

**Discussion by DanL78** [nathanmarlor/foxess_em#20] [Ideas]
Is it possible to add more than one site from solcast?

Due to the way solcast records a site you can only enter one side of a pitched roof per site, so having solar panels on both side of your roof (ie, west facing side of the roof and east facing side roof) have to be entered as seperate sites to get accurate data.

Unfortunately this means we can only record half the projected solar generation, ideally we'd need to add two sites and add the generation together to create one forecast. - if possible.

Thanks in advance

**Comment by nathanmarlor** [comment]
Thanks @DanL78 - have moved to an enhancement.

**Comment by nathanmarlor** [comment]
@DanL78 - implemented this today, please checkout the v1.4.x beta release :+1:
